code/game/jobs/job/mercenaries.dm

```diff
@@ -2,20 +2,96 @@
 	special_role = "Mercenary"
 	comm_title = "Merc"
 	faction = "Unknown Mercenary Group"
-	total_positions = 0
-	spawn_positions = 0
-	access = list(ACCESS_IFF_PMC, ACCESS_WY_PMC_GREEN, ACCESS_WY_PMC_ORANGE, ACCESS_WY_PMC_RED, ACCESS_WY_PMC_BLACK, ACCESS_WY_CORPORATE)
+	idtype = /obj/item/card/id
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
 	skills_type = /datum/skills/mercenary
 
 //Mercenary Heavy
 /datum/job/mercenaries/heavy
-	title = "Mercenary Enforcer"
+	title = "Mercenary Heavy"
+	paygrade = "MRC"
+	equipment = TRUE
+	access = list(ACCESS_IFF_MARINE, ACCESS_MARINE_COMMANDER, ACCESS_MARINE_LOGISTICS, ACCESS_MARINE_BRIDGE, ACCESS_MARINE_BRIG,
```

Comment:
If this is all-access, a define for it would be positive. There is also a function out there, such as the one synths use.
Defines are better, of course.
Same for the other instances of all-access.